src/haddock/libs/libutil.py

```diff
@@ -128,6 +128,31 @@ def remove_dict_keys(d: ParamMap, keys: Container[str]) -> ParamDict:
     return {k: deepcopy(v) for k, v in d.items() if k not in keys}
 
 
+def cpu_count() -> int:
+    """Count number of available CPU for the process.
+    
+    User suggestion, by https://github.com/EricDeveaud
+
+    Note: pid 0 == current process
+
+    FIXME: from python3.13, better to use os.process_cpu_count()
+
+    Returns
+    -------
+    process_ncores : int
+        Number of cores allocated to the process pid.
+    """
```

Comment:
shall we add a test for this new function

Reply:
I do not know how, because we need to know what setup is running the test